2020-05-19 00:09:56,044 ERROR: Exception on /reset_password_request [GET] [in D:\learning\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\learning\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\learning\microblog\app\routes.py", line 161, in reset_password_request
    form = ResetPasswordRequestForm()
NameError: name 'ResetPasswordRequestForm' is not defined
2020-05-19 00:10:12,611 ERROR: Exception on /reset_password_request [GET] [in D:\learning\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\learning\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\learning\microblog\app\routes.py", line 161, in reset_password_request
    form = ResetPasswordRequestForm()
NameError: name 'ResetPasswordRequestForm' is not defined
2020-05-19 00:22:15,915 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:49]
2020-05-19 00:22:26,926 ERROR: Exception on /reset_password_request [GET] [in D:\learning\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\learning\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\learning\microblog\app\routes.py", line 112, in reset_password_request
    form = ResetPasswordRequestForm()
NameError: name 'ResetPasswordRequestForm' is not defined
2020-05-19 00:24:36,865 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:49]
2020-05-19 00:24:48,517 ERROR: Exception on /reset_password_request [GET] [in D:\learning\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\learning\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\learning\microblog\app\routes.py", line 112, in reset_password_request
    form = ResetPasswordRequestForm()
NameError: name 'ResetPasswordRequestForm' is not defined
2020-05-19 00:27:02,524 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:49]
2020-05-19 00:27:18,250 ERROR: Exception on /reset_password_request [POST] [in D:\learning\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\learning\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\learning\microblog\app\routes.py", line 116, in reset_password_request
    send_password_reset_email(user)
NameError: name 'send_password_reset_email' is not defined
2020-05-19 00:31:25,914 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:49]
2020-05-19 00:31:41,937 ERROR: Exception on /reset_password_request [POST] [in D:\learning\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\learning\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\learning\microblog\app\routes.py", line 117, in reset_password_request
    send_password_reset_email(user)
  File "D:\learning\microblog\app\email.py", line 14, in send_password_reset_email
    sender=app.config['ADMINS'][0],
NameError: name 'app' is not defined
2020-05-19 00:32:33,924 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:49]
2020-05-19 00:32:44,994 ERROR: Exception on /reset_password_request [POST] [in D:\learning\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\learning\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\learning\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\learning\microblog\app\routes.py", line 117, in reset_password_request
    send_password_reset_email(user)
  File "D:\learning\microblog\app\email.py", line 16, in send_password_reset_email
    text_body=render_template('email/reset_password.txt',
NameError: name 'render_template' is not defined
2020-05-19 00:34:08,284 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:49]
2020-05-19 00:35:44,018 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:49]
2020-05-19 00:40:28,616 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:49]
2020-05-19 00:42:27,896 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:49]
2020-05-19 13:31:31,484 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:49]
2020-05-19 19:44:45,838 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 19:50:08,942 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 19:55:21,790 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 19:56:09,228 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:01:04,679 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:02:54,021 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
2020-05-19 20:04:30,259 INFO: Microblog startup [in D:\learning\microblog\app\__init__.py:51]
